gobblin-metastore/src/main/java/org/apache/gobblin/metastore/JobHistoryDataSourceProvider.java

```diff
@@ -17,24 +17,42 @@
 
 package org.apache.gobblin.metastore;
 
+import java.time.Duration;
 import java.util.Properties;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
 
 import org.apache.gobblin.configuration.ConfigurationKeys;
 import org.apache.gobblin.password.PasswordManager;
+import org.apache.gobblin.util.jdbc.DataSourceProvider;
+import org.apache.gobblin.util.jdbc.MysqlDataSourceUtils;
 
 
 /**
  * This class extends {@link org.apache.gobblin.util.jdbc.DataSourceProvider} with its own property keys.
  */
 public class JobHistoryDataSourceProvider extends org.apache.gobblin.util.jdbc.DataSourceProvider {
+  private static final Logger LOG = LoggerFactory.getLogger(DataSourceProvider.class);
```

Comment:
* JobHistoryDataSourceProvider
or you can use @Slf4j

Reply:
the other classes use logger factory so sticking with that although @sl4j also works.

Reply:
do you want to update the class name given to the factory, right?